lib/OnyxUtils.ts

```diff
@@ -107,10 +109,10 @@ function initStoreValues(keys: DeepRecord<string, OnyxKey>, initialKeyStates: Pa
     // We need the value of the collection keys later for checking if a
     // key is a collection. We store it in a map for faster lookup.
     const collectionValues = Object.values(keys.COLLECTION ?? {});
-    onyxCollectionKeyMap = collectionValues.reduce((acc, val) => {
-        acc.set(val, true);
+    onyxCollectionKeySet = collectionValues.reduce((acc, val) => {
+        acc.add(val as OnyxKey); // todo
```

Comment:
Instead of assertion here, let's validate that `collectionValues` is of type `string[]`, we could also just do this:
```ts
const collectionValues = Object.values(keys.COLLECTION ?? {}) as string[];
```